mlflow/tensorflow.py

```diff
@@ -64,7 +77,12 @@ def predict(self, df):
 
 
 def log_saved_model(saved_model_dir, signature_def_key, artifact_path):
-    """Log a Tensorflow model as an MLflow artifact for the current run."""
+    """Log a Tensorflow model as an MLflow artifact for the current run.
+
+    :param saved_model_dir: Directory where the exported tf model is saved.
+    :param signature_def_key: Which signature definition to use when loading the model again. See https://www.tensorflow.org/serving/signature_defs for details.
+    :param artifact_path: Path to where artifacts of the model will be saved.
```

Comment:
Suggestion: "Path to where..." -> "Path (within the artifact directory for the current run) to which..."